GenderPayGap.WebUI/Views/ManageOrganisations/OrganisationReportsTable.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@using GenderPayGap.Core.Classes
+@using GenderPayGap.WebUI.Models.Organisation
+@using GenderPayGap.Core
+@model GenderPayGap.WebUI.Models.Organisation.OrganisationReportsViewModel
+
+<table class="govuk-table">
+    <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Year</th>
+            <th scope="col" class="govuk-table__header">Reporting requirement</th>
+            <th scope="col" class="govuk-table__header">Report status</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        @{
+            foreach (KeyValuePair<int, OrganisationScopeForYear> scopeForYear in Model.OrganisationScopesByYear)
+            {
+                int reportingYear = scopeForYear.Key;
+                OrganisationScopeForYear organisationScopeForYear = scopeForYear.Value;
+
+                string formattedYearText = $"{reportingYear}/{(reportingYear + 1).ToTwoDigitYear()}";
+                
+                DateTime snapshotDate = organisationScopeForYear.GetSnapshotDate();
+                DateTime deadline = snapshotDate.AddYears(1).AddDays(-1);

```

Comment:
Could this go in the viewmodel too? (Instead of having a local var at all I mean - making `GetByReportingDeadlineText` take no arguments)